controller/update_test.go

```diff
@@ -112,32 +117,45 @@ func (s *UpdateControllerTestSuite) TestStartUpdateOk() {
 
 		// then
 		err := test.WaitWithTimeout(10 * time.Second).Until(func() error {
-			if int(*updateExecutor.NumberOfCalls) != 60 {
+			if int(*updateExecutor.NumberOfCalls) != 12 {
 				return fmt.Errorf("expeced number of calls 60 wasn't fullfiled - actual: %d", int(*updateExecutor.NumberOfCalls))
```

Comment:
Because the original logic was calling the executor for every namespace separately (which means five times per one tenant) - this new logic calls executor only once for one tenant and gives a list of env types to be updated - the update of the actual namespaces is managed by the executor